beginner_source/text_sentiment_ngrams_tutorial.py

```diff
@@ -1,23 +1,23 @@
 """
-Text Classification Tutorial
-============================
+Text Classification with TorchText
+==================================
 
-This tutorial shows how to use the text classification datasets,
-including
+This tutorial shows how to use the text classification datasets
+in ``torchtext``, including
 
 ::
 
    - AG_NEWS,
-   - SogouNews, 
-   - DBpedia, 
+   - SogouNews,
+   - DBpedia,
    - YelpReviewPolarity,
-   - YelpReviewFull, 
-   - YahooAnswers, 
+   - YelpReviewFull,
+   - YahooAnswers,
    - AmazonReviewPolarity,
    - AmazonReviewFull
 
-This example shows the application of ``TextClassification`` Dataset for
-supervised learning analysis.
+This example shows how to train a supervised learning algorithm on
```

Comment:
I would use `supervised learning model for`

Reply:
Changed wording in latest commit.